lib/ansible/module_utils/vmware.py

```diff
@@ -456,15 +669,19 @@ def push_file_to_guest(content, vm, username, password, src, dest, overwrite=Tru
     return result
 
 
-def run_command_in_guest(content, vm, username, password, program_path, program_args, program_cwd, program_env):
+def run_command_in_guest(
+    content, vm, username, password, program_path, program_args, program_cwd,
+    program_env
+):
 
     result = {'failed': False}
 
     tools_status = vm.guest.toolsStatus
     if (tools_status == 'toolsNotInstalled' or
-                tools_status == 'toolsNotRunning'):
+            tools_status == 'toolsNotRunning'):
         result['failed'] = True
-        result['msg'] = "VMwareTools is not installed or is not running in the guest"
+        result['msg'] = "VMwareTools is not installed"
+        result['msg'] = " or is not running in the guest"
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.